src/Symfony/Component/HttpKernel/Tests/HttpCache/EsiTest.php

```diff
@@ -102,7 +102,7 @@ public function testMultilineEsiRemoveTagsAreRemoved()
         $response = new Response('<esi:remove> <a href="http://www.example.com">www.example.com</a> </esi:remove> Keep this'."<esi:remove>\n <a>www.example.com</a> </esi:remove> And this");
         $this->assertSame($response, $esi->process($request, $response));
 
-        $this->assertEquals(' Keep this And this', $response->getContent());
+        $this->assertEquals(' Keep this And this', substr($response->getContent(), 24, -24));
```

Comment:
This looks like a BC break not suitable in a patch release as projects might use the Esi class directly.

Reply:
This is a security-related fix, I'd better break existing implems that does that so that they can adjust. (I also doubt this will hit anyone in practice :crossed_fingers: )

Reply:
then the boundary length should at least be a public constant, so that they don't have to hardcode this `24` everywhere